airflow-core/src/airflow/ui/src/i18n/locales/de/common.json

```diff
@@ -0,0 +1,78 @@
+{
+  "defaultToGraphView": "Graph-Ansicht als Standard",
+  "defaultToGridView": "Gitter-Ansicht als Standard",
+  "logout": "Abmelden",
+  "switchToDarkMode": "Zum Dunkelmodus wechseln",
+  "switchToLightMode": "Zum Hellmodus wechseln",
+  "timezone": "Zeitzone",
+  "user": "Benutzer",
+  "language": {
+    "chinese": "Traditionelles Chinesisch",
+    "english": "Englisch",
+    "german": "Deutsch",
+    "select": "Sprache wählen"
+  },
+  "nav": {
+    "home": "Start",
+    "assets": "Datenbestände",
```

Comment:
I was thinking about that yesterday, what "assets" could be and it is tricky, I like "Datenbestände" (the best I could come up with was "Datenobjekte"). I am curious what the feedback will be on this term in particular :)

Reply:
I also had a sleep over this. I assume the Grman edition of the legacy "Datasets" term is matching. Proposing to use "Datensets (Asstes)" now. Then we have the english/original technical term on the UI as well.

![image](https://github.com/user-attachments/assets/839a415c-aaba-43e9-a166-11e588d58360)